### PR TITLE
refactor: add npx to scripts execution

### LINKED ISSUE #13: fix test script to invoke asyncapi cli with npx
In https://github.com/derberg/python-mqtt-client-template/blob/main/package.json#L21 you need to modify the call of `asyncapi` command to use `npx` tool

example https://www.asyncapi.com/docs/tools/generator/usage#cli-usage-with-npx-instead-of-npm

### PR DESCRIPTION
**Pull request by thulieblack**
Add npx to run the generator script

resolves #13

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
         "@asyncapi/generator-react-sdk": "^0.2.25"
     },
     "scripts": {
-        "test:clean": "rimraf test/project/client.js",
-        "test:generate": "asyncapi generate fromTemplate test/fixtures/asyncapi.yml ./ --output test/project --force-write --param server=dev",
+        "test:clean": "npx rimraf test/project/client.js",
```
Comment (derberg) [inline package.json:20]:
```suggestion
        "test:clean": "npx rimraf@5.0.0 test/project/client.py",
```

File: package.json [hunk 2]
```diff
@@ -17,10 +17,10 @@
...
+        "test:generate": "npx asyncapi generate fromTemplate test/fixtures/asyncapi.yml ./ --output test/project --force-write --param server=dev",
         "test:start": "python test/project/test.py",
-        "test": "npm run test:clean && npm run test:generate && npm run test:start"
+        "test": "npx rimraf test/project/client.js && npx asyncapi generate fromTemplate test/fixtures/asyncapi.yml ./ --output test/project --force-write --param server=dev && python test/project/test.py"
```
Comment (derberg) [inline package.json:23]:
```suggestion
        "test": "npm run test:clean && npm run test:generate && npm run test:start"
```

File: package.json [hunk 3]
```diff
@@ -17,10 +17,10 @@
         "@asyncapi/generator-react-sdk": "^0.2.25"
     },
     "scripts": {
-        "test:clean": "rimraf test/project/client.js",
-        "test:generate": "asyncapi generate fromTemplate test/fixtures/asyncapi.yml ./ --output test/project --force-write --param server=dev",
+        "test:clean": "npx rimraf test/project/client.js",
+        "test:generate": "npx asyncapi generate fromTemplate test/fixtures/asyncapi.yml ./ --output test/project --force-write --param server=dev",
```
Comment (derberg) [inline package.json:21]:
```suggestion
        "test:generate": "npx asyncapi@1.5.14 generate fromTemplate test/fixtures/asyncapi.yml ./ --output test/project --force-write --param server=dev",
```

File: package.json [hunk 4]
```diff
@@ -17,10 +17,10 @@
...
         "test:start": "python test/project/test.py",
-        "test": "npm run test:clean && npm run test:generate && npm run test:start"
+        "test": "npx rimraf test/project/client.js && npx asyncapi generate fromTemplate test/fixtures/asyncapi.yml ./ --output test/project --force-write --param server=dev && python test/project/test.py"
     },
     "devDependencies": {
```
Comment (derberg) [inline package.json:25]: please remove and also do `npm i` again to refresh `package-lock.json` file

**Comment by thulieblack** [comment]
Done @derberg

File: package.json [hunk 5]
```diff
@@ -19,12 +19,9 @@
         "@asyncapi/generator-react-sdk": "^0.2.25"
     },
     "scripts": {
-        "test:clean": "rimraf test/project/client.js",
-        "test:generate": "asyncapi generate fromTemplate test/fixtures/asyncapi.yml ./ --output test/project --force-write --param server=dev",
+        "test:clean": "npx rimraf@5.0.0 test/project/client.py",
+        "test:generate": "npx asyncapi generate fromTemplate test/fixtures/asyncapi.yml ./ --output test/project --force-write --param server=dev",
```
Comment (derberg) [inline package.json:23]:
```suggestion
        "test:generate": "npx -p @asyncapi/cli@1.6.3 asyncapi generate fromTemplate test/fixtures/asyncapi.yml ./ --output test/project --force-write --param server=dev",
```

File: package.json [hunk 6]
```diff
@@ -19,12 +19,9 @@
         "@asyncapi/generator-react-sdk": "^0.2.25"
     },
     "scripts": {
-        "test:clean": "rimraf test/project/client.js",
-        "test:generate": "asyncapi generate fromTemplate test/fixtures/asyncapi.yml ./ --output test/project --force-write --param server=dev",
+        "test:clean": "npx rimraf@5.0.0 test/project/client.py",
```
Comment (derberg) [inline package.json:22]:
```suggestion
        "test:clean": "npx rimraf@5.0.0 test/project/client.py test/project/requirements.txt",
```

**Review by derberg — APPROVED** [review]

**Pull request merged**